{
    "debug": {
        "id": 186601
    },
    "sections": [
        {
            "id": "111240982",
            "content": [
                {
                    "testField": "data",
                    "testData": 1
                },
                {
                    "testField": "not data",
                    "testData": 2
                }
            ],
            "errors": [
                {
                    "message": "Fapi exception message here",
                    "kind": "resolver_error"
                }
            ],
            "type": "EngineTestSection"
        }
    ]
}